enzy_htp/structure/structure.py

```diff
@@ -225,6 +225,14 @@ def residue_mapper(self) -> Dict[Tuple[str, int], Residue]:
             result[residue.key()] = residue
         return result
 
+    @property
+    def residue_mapper_w_name(self) -> Dict[Tuple[str, int, str], Residue]:
```

Comment:
I would rename to `named_residue_mapper`. Always trying to keep method names short AND descriptive!

Reply:
nice English lol. I like this new name

Reply:
Wait this is not right. It sounds like the mapper is named but it is not. It is the keys have residue names in it.

Reply:
ChatGPT suggested residue_namedkey_mapper lol. Sounds good